src/components/mdc/Menu/Menu.svelte

```diff
@@ -33,26 +34,25 @@ const handleItemClick = url => {
     $goto(url)
   }
 }
+const handleItemKeydown = (e, url) => (e.code == 'Space' || e.code == 'Enter') && handleItemClick(url)
+const onMenuClose = () => !menu.open && (menuToggler = false)
```

Comment:
I'm a little confused about the purpose of the `menuToggler` variable. It's name sound like it's an active thing, but it seems to be a boolean for... indicating whether the menu should be open?

Okay, I think it's making more sense now. What if we changed this line as follows?
```suggestion
const closeMenu = () => {
  if (menu.open) {
    menuToggler = false
  }
}
```

We might not even need the `if` condition, but I don't know what the Menu component does if `open` is false and we set it to false again.

Reply:
menuToggler is a boolean used to set menu.open which makes the menu open or close. I will change menuToggler to menuOpen. If we could bind directly to menu.open we wouldn't need it. The problem was that clicking off the menu set menu.open to false but I couldn't communicate that back to menuToggler without circular statements.  After trying without the if I realized we need it for clicking on the menu itself.
`const closeMenu = () => {
  if (!menu.open) {
    menuToggler = false
  }
}`

Reply:
Just to clarify, the menu closes itself we the user clicks, focuses elsewhere. We are just trying to let the parent know to synchronize with the prop so that menuOpen is reset to false when we try to open it the second time.